src/components/DatePicker/index.jsx

```diff
@@ -86,10 +94,27 @@ export default class DatePickerComponent extends React.Component {
   }
 
   // Property initializer binds method to class instance
-  onChange = date => {
-    this.updateValue(date);
-    this.props.onChange(date);
+  onChange = value => {
+    const {
+      onChange,
+      name
+    } = this.props;
+
+    this.updateValue(value);
     this.hideError();
+
+    /*
+     * react-datepicker won't return a native DOM event
+     * on their onChange callback, so we need to create
+     * a fake one, because kununu-form-wrapper and other
+     * form libs will expect a DOM event object to process this field
```

Comment:
Nice description! (But since this is a public repo and others won't understand what kununu-form-wrapper is I would just remove that part)